meson.build

```diff
@@ -1,32 +1,24 @@
-project('x86-simd-sort', 'c', 'cpp',
+project('x86-simd-sort', 'cpp',
         version : '1.0.0',
         license : 'BSD 3-clause')
-cc = meson.get_compiler('c')
 cpp = meson.get_compiler('cpp')
-src = include_directories('./src')
-bench = include_directories('./benchmarks')
-utils = include_directories('./utils')
-tests = include_directories('./tests')
-gtest_dep = dependency('gtest', fallback : ['gtest', 'gtest_dep'])
-subdir('./tests')
+src = include_directories('src')
+bench = include_directories('benchmarks')
+utils = include_directories('utils')
+tests = include_directories('tests')
```

Comment:
These can all be assigned to one variable, using a single `include_directories()` invocation with an array.